HPXMLtoOpenStudio/resources/defaults.rb

```diff
@@ -3176,6 +3179,402 @@ def self.apply_pv_systems(hpxml_bldg)
     end
   end
 
+  # TODO
+  def self.get_id(prefix, objects, unit_num)
```

Comment:
Seems like a good idea. Can you apply this to other places in the code that use similar logic? A quick search shows `WeatherStation` and `GeothermalLoop`.

Reply:
Huh? I don't see it used for GeothermalLoop. I'm looking at [this](https://github.com/NREL/OpenStudio-HPXML/blob/resstock/HPXMLtoOpenStudio/resources/defaults.rb#L2302-L2306).

And [here](https://github.com/NREL/OpenStudio-HPXML/blob/resstock/HPXMLtoOpenStudio/resources/defaults.rb#L954) for WeatherStation.

Reply:
(Sorry, I wrote my comment in the past tense relative to changes I'm making based on your feedback.)

That's what I'm looking at for `WeatherStation`, too. So you want to change the id string here?

Reply:
Ah, got it. And yes, would be nice to set the WeatherStation id string via this method too. I realize it's slightly different because it's not an array, so if it's a pain, you can just forget it.

Reply:
I'm not totally sure what you're suggesting here -- modify the contents of the method as it is here?